kafka-connect-upload-source/src/main/java/org/radarbase/connect/upload/converter/axivity/AccelerationCwaBlockProcessor.kt

```diff
@@ -0,0 +1,20 @@
+package org.radarbase.connect.upload.converter.axivity
+
+import org.radarbase.connect.upload.converter.RecordLogger
+import org.radarbase.connect.upload.converter.TopicData
+import org.radarbase.connect.upload.converter.axivity.newcastle.CwaBlock
+import org.radarcns.connector.upload.axivity.AxivityAcceleration
+
+class AccelerationCwaBlockProcessor: CwaFileProcessorFactory.CwaBlockProcessor {
+    override fun processBlock(recordLogger: RecordLogger, block: CwaBlock, timeReceived: Double): List<TopicData> {
+        return (0 until block.numSamples)
+                .map { i -> TopicData("connect_upload_axivity_acceleration",
+                        AxivityAcceleration(
+                                block.startTime(),
```

Comment:
I am not too sure about setting the start time of the block for all records.
It makes sense based on what we discussed, although I am not sure if it is identical to what their CwaCsvInputStream was producing. If it is, then it is ok. Otherwise, I guess the rest of the computation and analysis they have based on their parser would produce different result than expected?